front/pages/w/[wId]/assistant/assistants.tsx

```diff
@@ -181,6 +181,7 @@ export default function PersonalAssistants({
           show={showDetails !== null}
           onClose={() => {
             setShowDetails(null);
+            void mutateAgentConfigurations();
```

Comment:
Can we gate this change behind a `shouldMutate` and only call this when the agent configuration has actually been mutated?